demos/button.html

```diff
@@ -147,6 +147,10 @@ <h1 class="mdc-typography--display2">Ripple Enabled</h1>
             <a href="javascript:void(0)" class="mdc-button">
               Link
             </a>
+            <button class="mdc-button">
```

Comment:
We wanted to keep link last since it's the only one that doesn't get disabled with the checkbox at the top of the demo page, right?

Reply:
I don't hold a strong opinion here, but this is a reasonable ask since that looks cleaner. Will move it.